web/src/components/ZWebsiteNav/ZWebsiteNav.vue

```diff
@@ -118,7 +118,7 @@
       z-lang-picker.z-website-nav__link(
         v-if='langItems',
         :lang='lang',
-        @close-nav="onClickBurger()"
+        @click-item="onClickBurger()"
```

Comment:
OK.
This is most common to pass the handler directly, not the call to it:
```suggestion
        @click-item="onClickBurger"
```